.github/workflows/database-migrations.yml

```diff
@@ -38,4 +38,7 @@ jobs:
           role-to-assume: arn:aws:iam::<ACCOUNT_ID>:role/<PROJECT_NAME>-github-actions
           aws-region: us-east-1
       - name: Run migrations
-        run: make release-run-database-migrations APP_NAME=${{ inputs.app_name }} ENVIRONMENT=${{ inputs.environment }}
+        run: |
+          echo "::group::Database Migrations"
+          make release-run-database-migrations APP_NAME=${{ inputs.app_name }} ENVIRONMENT=${{ inputs.environment }}
+          echo "::endgroup::"
```

Comment:
Looks great. I was thinking though, I wonder if this is grouped at too high of a level. Could we try grouping `Step 1. Update task definition without updating service` and `Step 2. Run "db-migrate" command` and see what it looks like?

so the logs then by default would be like:

```
 ./bin/run-database-migrations.sh app efc3827a549c05de2666d1e2c698609abc134eb2 dev
  ==================
  Running migrations
  ==================
  Input parameters
    APP_NAME=app
    IMAGE_TAG=efc3827a549c05de2666d1e2c698609abc134eb2
    ENVIRONMENT=dev
  
  Step 0. Check if app has a database
  
  ▶ Step 1. Update task definition without updating service

  ▶ Step 2. Run "db-migrate" command
```

which I think is still pretty readable but gives a bit more information, and allows the developer to expand the specific step they want to look at. Thoughts?

Reply:
@lorenyu I'll give that a try. It seems like it would make it easier for folks to read the information more relevant to what they're working on